setup.cfg

```diff
@@ -11,10 +11,10 @@ classifiers =
 [options]
 packages = find:
 install_requires =
-    numpy
+    numpy==1.23.5
```

Comment:
I would prefer this to be numpy>=1.23.5 if that is ok? Otherwise it is quite restrictive.

Reply:
@edbeeching there seems to be a problem with rllib and numpy 1.24.x (https://github.com/ray-project/ray/issues/31293#issuecomment-1379007369) that's why I set it to 1.23. Apparently there is a fix on ray's side, so once a newer ray version is deployed we can skip this again. But for now it seems to be still needed.